proto/nft/query.proto

```diff
@@ -11,27 +11,27 @@ option go_package = "github.com/irisnet/irismod/modules/nft/types";
 service Query {
     // Supply queries the total supply of a given denom or owner
     rpc Supply(QuerySupplyRequest) returns (QuerySupplyResponse) {
-      option (google.api.http).get = "/irismod/nft/supplies/{denom}";
+      option (google.api.http).get = "/irismod/nft/nfts/supplies/{denom}";
     }
 
     // Owner queries the NFTs of the specified owner
     rpc Owner(QueryOwnerRequest) returns (QueryOwnerResponse) {
-      option (google.api.http).get = "/irismod/nft/owners/{owner}";
+      option (google.api.http).get = "/irismod/nft/nfts/owners/{owner}";
```

Comment:
`nfts` means all `NF-token`,
not contains `owners` unless it means the owner of one NFT,  
`collections`,  `denoms`.